modDesc.xml

```diff
@@ -91,5 +103,11 @@
             <fr><![CDATA[Allemagne]]></fr>
```

Comment:
@Alagoss How do you say Germany and France in Dutch?

Reply:
Added this bit =P